packages/astro/src/core/create-vite.ts

```diff
@@ -21,6 +21,9 @@ import astroScriptsPlugin from '../vite-plugin-scripts/index.js';
 import astroScriptsPageSSRPlugin from '../vite-plugin-scripts/page-ssr.js';
 import { createCustomViteLogger } from './errors/dev/index.js';
 import { resolveDependency } from './util.js';
+import { astroContentServerPlugin } from '../content/vite-plugin-content-server.js';
```

Comment:
We've been using a pattern where a core module has an `index.ts` that re-exports its "public" API and anything that uses it imports from that module rather than reach into the internals. See `core/errors/` for example. Would be good to follow that pattern with content as well.